skimage/measure/_marching_cubes.py

```diff
@@ -130,9 +142,6 @@ def marching_cubes(volume, level, spacing=(1., 1., 1.),
     verts = np.asarray(verts)
     faces = np.asarray(faces)
 
-    # Calculate gradient of `volume`, then interpolate to vertices in `verts`
-    grad_x, grad_y, grad_z = np.gradient(volume)
-
```

Comment:
@JDWarner please double-check, but it looks like this was a leftover. I can recommend `pyflakes` to detect cases like this :)